Site/dataobjects/SiteAccount.php

```diff
@@ -858,6 +858,30 @@ protected function loadLoginSessions()
 			SwatDBClassMap::get('SiteAccountLoginSessionWrapper'));
 	}
 
+	// }}}
+	// {{{ public function loadByApiIdent()
+
+	public function loadByApiIdent($ident, SiteApiCredential $credential)
+	{
+		$this->checkDB();
+
+		$sql = sprintf(
+			'select account from Subscription
```

Comment:
There is no subscription table in the Site package. This code belongs in the Kos package.